iroh-dns-server/Cargo.toml

```diff
@@ -24,43 +24,38 @@ derive_more = { version = "1.0.0", features = [
     "from",
 ] }
 dirs-next = "2.0.0"
-futures-lite = "2.3.0"
-governor = "0.6.3"
+futures-lite = "2.5"
+governor = "0.6.3" #needs new release of tower_governor for 0.7.0
```

Comment:
I don't have any great way of doing this automatically or regularly. Im open to suggestions here on how to keep track of these other than to just manually revisit every so often and do this whole dance.